src/routes/docs/tooling/appwriter/+page.markdoc

```diff
@@ -46,11 +46,12 @@ Press `Fn + Home` to change the RGB backlight effect and cycle through all the l
 
 # How to get the Appwriter {% #how-to-get-the-appwriter %}
 
-The Appwriter is currently exclusive to Appwrite events and giveaways.
-To get your hands on an Appwriter, look out for events & giveaways on our social media
+The Appwriter is available for preorder in the [Appwrite Store](https://appwrite.store/products/preorder-the-appwriter).
+
+The Appwriter is also available from Appwrite events and giveaways.
```

Comment:
This sentence feels a bit of. Maybe we can say: "You can also partake in our giveaways and win the Appwriter."